module/docs/entra-powershell-beta/DirectoryManagement/New-EntraBetaAdministrativeUnitMember.md

```diff
@@ -51,7 +51,7 @@ The `New-EntraBetaAdministrativeUnitMember` cmdlet creates a Microsoft Entra ID
 
 Currently only Microsoft Entra ID groups are supported to create administrative unit members.
 
-For information about creating dynamic groups, see Using attributes to create advanced rules (<https://azure.microsoft.com/documentation/articles/active-directory-accessmanagement-groups-with-advanced-rules/>).
+For information about creating dynamic groups, see [Using attributes to create advanced rules](/entra/identity/users/groups-dynamic-membership).
```

Comment:
Let's use absolute URIs instead of relative URIs since we load these docs on PowerShell `Get-Help` command